hadoop-ozone/dist/src/main/smoketest/balancer/testBalancer.robot

```diff
@@ -67,7 +68,7 @@ Run Container Balancer
 Wait Finish Of Balancing
     ${result} =             Execute                         ozone admin containerbalancer status
                             Should Contain                  ${result}             ContainerBalancer is Running.
-                            Wait Until Keyword Succeeds      3min    10sec    ContainerBalancer is Not Running
+                            Wait Until Keyword Succeeds      6min    10sec    ContainerBalancer is Not Running
```

Comment:
What is the reason behind doubling the timeout here?

Reply:
It’s not exactly a timeout. We expect the balancing to complete within 6 minutes, but if it finishes earlier, we will proceed with the test.

Reply:
I see, but earlier the tests seem to have been working fine within 3mins. Did you observe some cases where it took more than 3 mins to complete the balancing during testing?

Reply:
Yes, in case of the EC container, it takes more than 5 minutes.

Reply:
Okay, Changes LGTM.